Text Formatting.html

```diff
@@ -540,19 +543,20 @@ <h3><a name="Footprint">Binary footprint</a></h3>
 
 <p>
 <code>basic_format_args</code> can be implemented as an array of tagged unions.
-Tags that indicate types of aguments can be combined and passed into a
+Tags that indicate types of arguments can be combined and passed into a
 formatting function as a single integer if the number of arguments is small.
 Since argument types are known at compile time this can be an integer
-constant and there will be no code generated to compute it at runtime, only to
+constant and there will be no runtime code generated to compute it, only to
 store according to calling conventions.
 </p>
+TODO: Not clear to me what the last sentence above means
```

Comment:
Consider the following example (https://godbolt.org/g/h2FohJ):

```
fmt::print("{}", 42, 4.2);
```

The argument types are `int` and `double` and their type tags are combined in a single integer and stored in register `edx`:

```
mov edx, 130
```

where `130 = 0x82 = 0x8 << 4 + 0x2`, `8` is a tag of `double` and `2` is a tag of `int`. What I'm trying to say is that this integer `0x82` representing all argument types is computed at compile time, not runtime, and just need to be stored in a register or stack (depending on calling conventions) which results in small binary code. Any ideas on how to make this section more clear are welcome.